src/ISA/ISA/ArcTypes/ArcTable.fs

```diff
@@ -621,20 +621,17 @@ type ArcTable =
         ]
         |> String.concat "\n"
 
+    member this.structurallyEquivalent (other: ArcTable) =
+        let n = this.Name = other.Name
+        let headers = Aux.compareSeq this.Headers other.Headers
+        let values = Aux.compareSeq (this.Values |> Seq.sortBy (fun x -> x.Key)) (other.Values |> Seq.sortBy (fun x -> x.Key))
+        n && headers && values
+
     // custom check
     override this.Equals other =
```

Comment:
Add identity Equivalence as alternative, explicit method.

https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/how-to-test-for-reference-equality-identity

Reply:
Done! 👍